pkgs/tools/security/enum4linux-ng/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, impacket
+, ldap3
+, pyyaml
+, samba
+, stdenv
```

Comment:
As a result of @SuperSandro2000's suggestion above, this can be removed as well.

```suggestion
```

Reply:
Removed, thanks.